seed/models/analysis_input_files.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+from seed.models import Analysis
+
+
+class AnalysisInputFile(models.Model):
+    """
+    The AnalysisInputFile is a file used as input for an analysis.
+
+    For example, if running an analysis on multiple properties, it might be a
+    CSV containing data collected from each property.
+    """
+    DEFAULT = 1
+
+    CONTENT_TYPES = (
+        (DEFAULT, 'default'),
```

Comment:
What's meant by `default`? I was expecting to see something BSyncr-specific. (Same question for AnalysisOutputFile)

Reply:
I'm not too worried about this, I was thinking I'd just replace this once I start working on integrating with BSyncr since I don't know what I want to call the files for now